### PR TITLE
🎨   Implemented a multiple input block

### PR DESCRIPTION
**Pull request by conceptacherono**
# Description

As a user you should be able to add a block that accepts multiple inputs.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# Screenshot (optional)
multiple-input-block

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: libs/features/convs-mgr/stories/blocks/library/multiple-input-message-block/src/lib/components/multiple-input-block/multiple-input-block.component.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+                  <div class="inputlist">
+                    <input [formControlName]="fIndex">
+                    <button  (click)="removeList(fIndex, groupIndex)" class="listgone"><i class="far fa-trash-alt"></i></button>
+                  </div>
+                </div>
+              </div>
+            </span>
+  
+            <button (click)="addNewOption(groupIndex)" mat-flat-button class="add-button">
+              Add Option
```
Comment (ReaganCn) [inline libs/features/convs-mgr/stories/blocks/library/multiple-input-message-block/src/lib/components/multiple-input-block/multiple-input-block.component.html:24]: Use multi-lang

File: libs/features/convs-mgr/stories/blocks/library/multiple-input-message-block/src/lib/components/multiple-input-block/multiple-input-block.component.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            </button>
+          </div>
+        </div>
+      </div>
+    </form>
+
+    <button (click)="addNewOptionGroup()" mat-flat-button class="add-button">
+      New List
```
Comment (ReaganCn) [inline libs/features/convs-mgr/stories/blocks/library/multiple-input-message-block/src/lib/components/multiple-input-block/multiple-input-block.component.html:32]: Also here. Use multi-lang

File: libs/model/convs-mgr/stories/blocks/messaging/src/lib/multiple-input-message-block.interface.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { StoryBlock } from "@app/model/convs-mgr/stories/blocks/main";
+
+export interface MultipleInputMessageBlock extends StoryBlock
```
Comment (ReaganCn) [inline libs/model/convs-mgr/stories/blocks/messaging/src/lib/multiple-input-message-block.interface.ts:3]: Explain what a multiple input message block does

File: libs/model/convs-mgr/stories/blocks/messaging/src/lib/multiple-input-message-block.interface.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+{
+  message?: string;
+
+  defaultTarget?: string;
+  
+  /** Response options */
+  options?: MultipleOptions[];
+
+  // Implement generic tags
+  tag?: string;
```
Comment (ReaganCn) [inline libs/model/convs-mgr/stories/blocks/messaging/src/lib/multiple-input-message-block.interface.ts:13]: What does `tag` and `context` do???

**Review by ReaganCn — CHANGES_REQUESTED** [review]
Please check my comments. Otherwise, good work

File: libs/model/convs-mgr/stories/blocks/messaging/src/lib/multiple-input-message-block.interface.ts (hunk 4 above)
Comment (conceptacherono) [reply libs/model/convs-mgr/stories/blocks/messaging/src/lib/multiple-input-message-block.interface.ts:18]: I have implemented these changes

**Review by ReaganCn — APPROVED** [review]

**Pull request merged**